src/classes/GE_PaymentServices.cls

```diff
@@ -54,142 +48,142 @@ public with sharing class GE_PaymentServices {
         REFUNDISSUED
     }
 
-    public String getEndpoint(ElevateEndpoints endpoint) {
-        if (endpoint === ElevateEndpoints.TOKENIZE) {
-            return '/v1/payments/public/tokens';
+    /***
+     * @description Payment Services configuration
+     */
+    @TestVisible
+    private PS_IntegrationServiceConfig.Service configService {
+        get {
+            if (configService == null) {
+                configService = new PS_IntegrationServiceConfig.Service();
+            }
+            return configService;
         }
+        set;
+    }
 
-        if (endpoint === ElevateEndpoints.PURCHASE) {
-            return '/v1/payments/verified/purchase';
+    /**
+     * @description Checks whether an org is a customer of Elevate Payment Services.
+     *              After onboarding within Elevate, these configuration key-value
+     *              pairs should exist in the org.  If this required set of keys is not
+     *              in the org, NPSP cannot load the Elevate SDK or successfully make
+     *              callouts to Elevate services.
+     */
+    public static Boolean isElevateCustomer {
+        get {
+            PS_IntegrationServiceConfig ps = new PS_IntegrationServiceConfig();
```

Comment:
I recommend storing a static instance of this getter to avoid re-instantiating PS_IntegrationServiceConfig() every time. It does look like the PS_IntegrationServiceConfig.config class is static, so I'm pretty sure this won't cause additional queries, but I do think that avoiding a new instantiation on multiple calls to this method will improve performance, even if just slightly.

Reply:
@ptewson-sfdo , @force2b : The `isElevateCustomer` is unchanged. I would keep it as is. Let me know.

Reply:
I could go either way (or we could also store a static instance of the PS_IntegrationServiceConfig class).... Mike's suggestion is a good one, but I also think the impact is probably minimal. Given that there's a complex QA process for this work and a tightening window, I'm maybe more on Radmila's side to minimize changes where the impact isn't thought to be high.

Reply:
@kenlewis-sfdo  has very valuable comments how to improve the class and initialize class variables. It is most beneficial to apply the change in a new tech debt/refactoring WI.